.github/scripts/run_with_env_secrets.py

```diff
@@ -83,9 +87,9 @@ def main():
             { os.environ.get('GPU_FLAG', '') } \
             -v "{ os.environ.get('GITHUB_WORKSPACE', '') }/{ os.environ.get('REPOSITORY', '') }:/{ os.environ.get('REPOSITORY', 'work') }" \
             -v "{ os.environ.get('GITHUB_WORKSPACE', '') }/test-infra:/test-infra" \
-            -v "{ os.environ.get('RUNNER_ARTIFACT_DIR', '') }:/artifacts" \
-            -v "{ os.environ.get('RUNNER_DOCS_DIR', '') }:/docs" \
-            -v "{ os.environ.get('RUNNER_TEST_RESULTS_DIR', '') }:/test-results" \
+            -v "{ os.environ.get('RUNNER_TEMP', '') }/artifacts:/artifacts" \
+            -v "{ os.environ.get('RUNNER_TEMP', '') }/docs:/docs" \
+            -v "{ os.environ.get('RUNNER_TEMP', '') }/test-results:/test-results" \
```

Comment:
Let's define these `RUNNER_` env vars in setup-rocm.yml similar to https://github.com/pytorch/test-infra/blob/853a7f2216767e97b894574163cfba8e392e333e/.github/actions/setup-linux/action.yml#L60, so we don't have to change the code here, since it's commonly used by both ROCm and non-ROCm workflows.